src/StoreKeeper/WooCommerce/B2C/Commands/SyncWoocommerceSingleProduct.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace StoreKeeper\WooCommerce\B2C\Commands;
+
+use Exception;
+use StoreKeeper\WooCommerce\B2C\Exceptions\BaseException;
+use StoreKeeper\WooCommerce\B2C\Imports\FullProductImportWithSelectiveIds;
+use StoreKeeper\WooCommerce\B2C\Tools\TaskHandler;
+
+class SyncWoocommerceSingleProduct extends SyncWoocommerceProductPage
+{
+    /**
+     * @return mixed|void
+     *
+     * @throws Exception
+     */
+    public function execute(array $arguments, array $assoc_arguments)
+    {
+        if ($this->prepareExecute()) {
+            if (array_key_exists('storekeeper_id', $assoc_arguments)) {
+                $this->runSync($assoc_arguments);
+            } else {
+                throw new BaseException('No product id provided');
+            }
+        }
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function runSync($assoc_arguments): void
+    {
+        $import = new FullProductImportWithSelectiveIds($assoc_arguments);
```

Comment:
Looks like all is implemented in ProductImport, why are you using FullProductImportWithSelectiveIds? Maybe better to add getter+setter?

```suggestion
        $import = new ProductImport($assoc_arguments);
        $import->setSyncProductVariations(true);
```